### ISSUE TITLE
Perform general a11y testing on site

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/wecount.inclusivedesign.ca#609]
It would be prudent to run some general accessibility testing on the site to ensure, for example, that it passes WCAG 2.0 AA (the current AODA requirement) if not a more stringent set of requirements. Effectively this work is an accessibility audit. Separate issues could be filed to compile any examples of failing criteria.

## Describe the solution you'd like

An in-depth accessibility audit of the website code, tracking any issues/failures in separate GitHub issues.

## Describe alternatives you've considered

N/A

## Additional context

The map component may or may not be included in this review, though any issues identified with it should be filed in the map's repo: https://github.com/inclusive-design/covid-data-monitor
